onnxscript/function_libs/torch_aten/graph_building.py

```diff
@@ -486,12 +481,17 @@ def to_model_proto(self, opset_version: int) -> onnx.ModelProto:
             function_proto_list.append(onnx_function.to_function_proto())
         onnx_model.functions.extend(function_proto_list)
         onnx_model = onnx.shape_inference.infer_shapes(
-            onnx_model, check_type=True, strict_mode=False
+            onnx_model, check_type=True, strict_mode=False, data_prop=True
         )
         try:
             onnx.checker.check_model(onnx_model, full_check=True)
-        except onnx.checker.ValidationError as e:
+        except (onnx.checker.ValidationError, onnx.shape_inference.InferenceError) as e:
```

Comment:
I think it is a bad thing that we rule out more and more checker errors. Are we waiting for checker to be updated?

Reply:
Yes there are shape inference errors to fix: https://github.com/onnx/onnx/issues/4986. We are always turning warnings into checker errors here. It's just some errors are emitted as shape inference errors and we didn't catch them.

Reply:
Could you add a comment so that we don't forget to enable it when it's fixed?

Reply:
I am inclined to always ignore them because this will allow a model to be exported. We will catch and fix any errors in our tests.